src/Voron/Impl/Paging/TempPagerTransaction.cs

```diff
@@ -30,6 +30,26 @@ public void Dispose()
             get; set;
         }
 
+        public long TotalEncryptionBufferSize
+        {
+            get
+            {
+                var cryptoTransactionStates = CryptoPagerTransactionState;
+                if (cryptoTransactionStates == null)
+                {
+                    return 0L;
+                }
+
+                var total = 0L;
```

Comment:
Can we remember this here as well?

Reply:
remember what?

Reply:
Remember the value, I meant. So we don't have to re-compute it at all times

Reply:
You can't just remember it, it is been modified constantly, the majority of the computation is done per CryptoState and that is already been computed on the fly.
Computing the total for CryptoPagerTransactionState will mean hiding it too and I don't see the gain here it is already a tiny price to pay for a very expensive operation.